lib/xboxrt/vcruntime/framehandler.s

```diff
@@ -0,0 +1,12 @@
+// int __cdecl __CxxFrameHandler3 (PEXCEPTION_RECORD, PEXCEPTION_REGISTRATION, _CONTEXT *, void *);
+// Additionally, a function information table is passed in the eax register
+.globl ___CxxFrameHandler3
+___CxxFrameHandler3:
+    pushl %eax
+    pushl 20(%esp)
+    pushl 20(%esp)
+    pushl 20(%esp)
+    pushl 20(%esp)
+    calll _CxxFrameHandlerVC8
```

Comment:
Question for my own curiosity: why the l suffix on the `call`?

Reply:
No particular reason.

Reply:
I'd prefer it without the suffix then, looks weird to me